ports/servoshell/tests/mod.rs

```diff
@@ -0,0 +1,6 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#[cfg(test)]
+mod servoshell_test;
```

Comment:
I think you can just include the contents of `servoshell_test.rs` here.

Reply:
done in [fab229d](https://github.com/servo/servo/pull/30238/commits/fab229d33f3b9003c8351cfae11a5c22187a2254)